stock_customer_deposit_elaboration/__manifest__.py

```diff
@@ -0,0 +1,20 @@
+# Copyright 2024 Moduon Team S.L.
+# License LGPL-3.0 or later (https://www.gnu.org/licenses/lgpl-3.0)
+
+{
+    "name": "Stock Customer Deposit Elaboration",
+    "summary": "Glue module betwen stock_customer_deposit and sale_elaboration",
+    "version": "16.0.1.0.0",
+    "development_status": "Alpha",
+    "category": "Inventory/Delivery",
+    "website": "https://github.com/OCA/stock-logistics-workflow",
+    "author": "Moduon, Odoo Community Association (OCA)",
+    "maintainers": ["EmilioPascual", "rafaelbn"],
+    "license": "LGPL-3",
+    "application": False,
+    "installable": True,
```

Comment:
issue: glue modules should auto-install

```suggestion
    "auto_install": True,
```